### ISSUE TITLE
[Docs] config missing backup settings

### ISSUE THREAD

**Issue by SlavikCA** [immich-app/immich#13997]
### The bug

Looking at the example of the config file here:
https://github.com/immich-app/immich/blob/main/docs/docs/install/config-file.md?plain=1

It is missing recently added backup settings.

I think this is the default, which needs to be added:
```
    "backup": {
      "database": {
        "enabled": true,
        "cronExpression": "0 02 * * *",
        "keepLastAmount": 14
      }
    },
```

### Version of Immich Server

v120.1

**Comment by SlavikCA** [comment]
Wow! Super quick response.

**Comment by zackpollard** [comment]
> Wow! Super quick response.

I was already in the area :laughing:

**Comment by zackpollard** [comment]
Fixed by #13996
